### ISSUE TITLE
⌊_⌋ (Relation.Nullary.Decidable) is untouchable

### ISSUE THREAD

**Issue by andreasabel** [agda/agda-stdlib#917]
Very recently, `⌊_⌋` was renamed to `isYes`.  I strongly recommend to abstain from this change.  I consider the identifier `⌊_⌋` sacrosanct.  It has been around for around a decade and appears in (I guess many) paper using Agda.

```
Warning: ⌊_⌋ was deprecated in v1.2.
Please use isYes instead.
when scope checking ⌊ b ≟ b' ⌋
...
Warning: ⌊_⌋ was deprecated in v1.2.
Please use isYes instead.
when scope checking ⌊ i ≟ j ⌋
...
Warning: ⌊_⌋ was deprecated in v1.2.
Please use isYes instead.
when scope checking ⌊ i ≤? j ⌋
```

**Comment by JacquesCarette** [comment]
Now that Andreas mentions it, I find myself agreeing that this change is a regression. There's too much inertia already built in to that name.

**Comment by MatthewDaggitt** [comment]
Fair enough, thanks for the heads up!

**Comment by andreasabel** [comment]
PR see #919.

**Comment by MatthewDaggitt** [comment]
Closed with e8927dd
